tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -873,8 +874,15 @@ LogicalResult LowerHLOToSharedLibrary(ModuleOp m,
 namespace tensorflow {
 
 Status ConvertTF2MlirHlo(mlir::ModuleOp module_op) {
-  mlir::PassManager pm(module_op.getContext());
+  mlir::DefaultTimingManager tm;
+  mlir::applyDefaultTimingManagerCLOptions(tm);
+  // Records elapsed time for each pass in the passpipe
+  tm.setEnabled(true);
+  mlir::TimingScope timing = tm.getRootScope();
```

Comment:
any pass running slow?

Reply:
Not in current UT tests. Just added in case necessary.